### ISSUE TITLE
Test262 properties outdated

### ISSUE THREAD

**Issue by andreabergia** [mozilla/rhino#1764]
I've noticed that the test262.properties file is outdated today. The diffs are surprising though:

```diff
-     from/calling-from-valid-1-noStrict.js non-strict Spec pretty clearly says this should be undefined
+     from/calling-from-valid-1-noStrict.js {compiled-non-strict,interpreted-non-strict} Spec pretty clearly says this should be undefined
```

So I am not sure if just rerunning and commiting is the right fix. I guess this might be related to the optimization level of https://github.com/mozilla/rhino/pull/1754 ?

**Comment by gbrail** [comment]
Yes, re-running and re-committing is the right fix, with a simple PR.
However, it might be worth looking at that PR -- there is a lot of complex
logic in there for calculating how to report on the test failures in the
four remaining test modes (non strict, strict, interpreted, and optimized)
and perhaps I missed some logic -- I think that both cases are the same,
but previously we must have consolidated them into one "non-strict" case
and now we don't.

**Comment by rbri** [comment]
Can we close this?

**Comment by gbrail** [comment]
I think so.

FWIW, for me, on three different OSes, even forcing Java 11 for the whole build, I can't use "updateTest262properties" without seeing a whole bunch of changes to that file that aren't relevant, so I made my last update manually. I wonder if anyone else has the same problem or if I'm doing something wrong.

**Comment by rbri** [comment]
Works for me with java 11.

**Comment by p-bakker** [comment]
Does this case need to remain open @andreabergia ?

**Comment by andreabergia** [comment]
Nope, sorry, it's fine now.
